src/component/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface HeaderProps {
+  todos: Todo[],
+  addTodo: () => void,
+  query: string,
+  setQuery: (query: string) => void,
+}
+
+export const Header: React.FC<HeaderProps> = ({
+  todos,
+  addTodo,
+  query,
+  setQuery,
+}) => {
+  const handleSubmit = (event:React.SyntheticEvent) => {
+    event.preventDefault();
+    addTodo();
```

Comment:
You need to pass the query as a parameter to the addTodo function. Replace this line with: addTodo(query);